modules/serverless-open-telemetry.adoc

```diff
@@ -0,0 +1,149 @@
+// Module is included in the following assemblies:
+//
+// * serverless/serverless-tracing.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-open-telemetry_{context}"]
+= Using {DTProductName} to enable distributed tracing
+
+{DTProductName} is made up of several components that work together to collect, store, and display tracing data. You can use {DTProductName} with {ServerlessProductName} to monitor and troubleshoot serverless applications.
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+* You have not yet installed the {ServerlessOperatorName} and Knative Serving. These must be installed after the {DTProductName} installation.
+* You have installed {DTProductName} by following the {product-title} "Installing distributed tracing" documentation.
+* Install the OpenShift (`oc`) CLI.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. Create an `OpenTelemetryCollector` custom resource (CR):
++
+.Example OpenTelemetryCollector CR
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: cluster-collector
+  namespace: tracing-system
+spec:
+  mode: deployment
+  config: |
+    receivers:
+      zipkin:
+    processors:
+    exporters:
+      jaeger:
+        endpoint: jaeger-all-in-one-inmemory-collector-headless.tracing-system.svc:14250
+        tls:
+          ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+      logging:
+    service:
+      pipelines:
+        traces:
+          receivers: [zipkin]
+          processors: []
+          exporters: [jaeger, logging]
+----
+
+. Verify that you have two pods running in the namespace where {DTProductName} is installed:
++
+[source,terminal]
+----
+$ oc get pods -n <namespace>
```

Comment:
Jus ta heads up: In this command we use `<namespace>` but in the other commands we use a concrete one `tracing-system`. Maybe better to use `tracing-system` here as well?